loadlsststack.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+LSSTSW_DIR="$(dirname "$(pwd)")/lsstsw"
+
+GCP_BUCKET="gs://eups-lsstsw-cache"
+
+TMP_FOLDER="/tmp/lsstswcache"
+
+fail() {
+  echo "$1" >&2
+  exit 1
+}
+
+[[ $# -eq 0 ]] && fail "No tag was passed" 
+# shellcheck disable=SC2054 disable=SC2034
+OPTS=$(getopt -o '' -l tag: -- "$@")
+eval set -- "$OPTS"
+while true
+do
+  case "$1" in
+    --tag)             DATE_TAG=$2             ; shift 2 ;;
+    --) shift; break;;
+    *) [[ "$*" != "" ]] && fail "Unknown option: $1"
+       break;;
+  esac
+done
```

Comment:
This seems rather baroque.  If the tag is required, why not just make it `$1` and skip all the parsing?

Also the shellcheck complaints being disabled don't seem like they should be triggered by the listed code.